cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityUserAndRoleManagementTest.scala

```diff
@@ -53,15 +67,58 @@ class SecurityUserAndRoleManagementTest extends DocumentingTest with QueryStatis
           resultTable()
         }
       }
+      section("Changing the logged in users password", "administration-security-users-alter-password") {
+        p("Users can be change their own password using `ALTER CURRENT USER SET PASSWORD`, " +
+          "the old password is required in addition to the new. " +
+          "This will both change the password and set the `CHANGE NOT REQUIRED` flag.")
+        query("ALTER CURRENT USER SET PASSWORD FROM 'abc123' TO '123xyz'", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          // Can't pull on the result since the test is run with auth disabled, which is not allowed for this command
+        }
+        note( () => {
+          p("This command cannot be run with auth disabled.")
+        })
+      }
+      section("Deleting users", "administration-security-users-drop") {
+        p("Users can be deleted using `DROP USER`.")
+        query("DROP USER jake", ResultAssertions((r) => {
+          assertStats(r, systemUpdates = 1)
+        })) {
+          p("Nothing is returned from this query, except the count of system database changes made.")
+          resultTable()
+        }
+        p("When a user has been deleted, it will no longer appear on the list provided by `SHOW USERS`.")
+        query("SHOW USERS", assertNodesShown("User", column = "user")) {
+          resultTable()
+        }
+        p("This command is optionally idempotent, with the default behavior to throw an exception if the user does not exists. " +
+          "Appending `IF EXISTS` to the command will ensure that no exception is thrown and nothing happens should the user not exist.")
+        query("DROP USER jake IF EXISTS", ResultAssertions( r => {
+          assertStats(r, systemUpdates = 0)
+        })) {}
+      }
     }
     section("Role Management", "administration-security-roles") {
+      initQueries("CREATE USER jake SET PASSWORD 'abc123' CHANGE NOT REQUIRED")
```

Comment:
Seems like my last bug-fix worked, now you can run initQueries like this! :-)

Reply:
It was very nice since the last thing I do in the user management is dropping the user :P

Reply:
Sadly it does not always work (sometimes I get weird test failures...

Ex:
Adding `initQueries("CREATE USER user1 SET PASSWORD 'abc'", "CREATE USER user2 SET PASSWORD 'abc'", "CREATE USER user3 SET PASSWORD 'abc'", "CREATE ROLE role1", "CREATE ROLE role2")` to the start of Assigning roles to users section breaks both granting and revoking role myrole to/from jake....